client/src/modules/stock/exit/exit.js

```diff
@@ -215,8 +215,12 @@ function StockExitController(
     vm.movement.description = $translate.instant(mapExit[exitType.label].description);
     vm.stockForm.store.clear();
     vm.resetEntryExitTypes = false;
-
     vm.overconsumption = [];
+
+    // reload inventories only if exit type has been selected previously
```

Comment:
This comment does not seem to match what you do in the "if (vm.depot)" block?

Reply:
Old code... 👍🏽